src/controllers/EventsController.js

```diff
@@ -1,9 +1,177 @@
 // @flow
 
+import moment from 'moment';
 import Controller from './Controller';
+import CoreController from '../lib/CoreController';
+import route from '../decorators/route';
+import httpVerb from '../decorators/httpVerb';
+
+import v1Api from '../views/api_v1';
+import logger from '../lib/logger';
 
 class EventsController extends Controller {
+  _aliveInterval: ?string;
+  _lastMessage: ?Date;
+  _socket: ?Object;
+
+  _cleanup() {
+    try {
+      if (this._socket) {
+        this._socket.close();
+        this._socket = null;
+      }
+
+      if (this.response && this.response.socket) {
```

Comment:
Hmmm... I think we might need to change how Controllers are constructed.  We should call `new` on the controller for reach request -- otherwise the `this.response` wouldn't work when multiple requests occur at the same time.